docs/docs/recipes/manage-users/on-admin-console-page.mdx

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_label: Manage on Admin Console page
+sidebar_position: 1
+---
+
+# Manage users on the Admin Console page
+
+import GotoUserManagement from './fragments/_goto-user-management.md';
+
+## List users
+
+<GotoUserManagement />
+
+There are three columns in the table:
+
+- **User**: The user info includes avatar, name (for display), and username (for sign-in).
```

Comment:
I can hardly distinguish username and name from the UI until I see `Unnamed`. Maybe give more detailed instructions.

Reply:
Maybe too detailed?

Reply:
We discussed this on [slack](https://silverhand-io.slack.com/archives/C0243P3E4KS/p1656039242323229) this morning, and the user column might show email, phone or username, and users' full name. Please align this before merging.

Reply:
Updated. PTAL